### ISSUE TITLE
Missing Node in Pretty Printer

### ISSUE THREAD

**Issue by SeeBasTStick** [hhu-stups/prob-issues#81]
The Node ```AOpSubstitution``` is missing in the pretty-printer causing wrong results.

**Comment by leuschel** [comment]
Do you have an example showing the issue?

**Comment by SeeBasTStick** [comment]
consider the following example:

```
MACHINE M1
VARIABLES x
INVARIANT x : NAT
INITIALISATION x := 1
OPERATIONS 
  Inc(amount, multiplier) = PRE amount : NAT & multiplier  > 1  THEN x := x + 1 END
END//MACHINE
```
```
MACHINE M2

INCLUDES M1


OPERATIONS 
  Foo = Inc(2,5)
END//MACHINE
```

Now consider the following code fragment:
```
BParser bParser= new BParser(pathToM2.toString());

Start start = bParser.parseFile(pathToM2.toFile(), false);

PrettyPrinter prettyPrinter = new PrettyPrinter();
prettyPrinter.caseStart(start);

System.out.println(prettyPrinter.getPrettyPrint());
```

The resulting pretty print is:
```
MACHINE M2
INCLUDES M1
OPERATIONS
Foo = Inc25
END
```

As the ```AOpSubstitution``` Node has no implementation in the pretty printer, the standard ```.toString()``` method of the parent class is utilized.

**Comment by favu100** [comment]
This issue is fixed by commit 7682e2763ae6223d27768ec87371da599594bbd0 in probparsers
